tests/test_editable.py

```diff
@@ -0,0 +1,37 @@
+import subprocess
+
+import pytest
+from virtualenv import cli_run
+
+from editables import build_editable
+
+
+def build_project(to, structure):
+    to.mkdir(exist_ok=True, parents=True)
+    for name, content in structure.items():
+        path = to / name
+        if isinstance(content, str):
+            path.write_text(content)
+        else:
+            build_project(path, content)
+
+
+def test_editable_expose_hide(tmp_path, capfd):
```

Comment:
here's an automated test to check this works would be wheel installed into a python environment

Reply:
Thanks! This was high priority for me so much appreciated.